speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/SQLDatabaseStorage.cpp

```diff
@@ -141,7 +142,7 @@ void SQLDatabaseStorage::CreateTables()
             "CREATE TABLE IF NOT EXISTS Environment (\n"
             "    environment_id  INT             NOT NULL AUTO_INCREMENT,\n"
             "    filename        VARCHAR(255)    NOT NULL,\n"
-            "    version         DATETIME(0)     NOT NULL,\n"
+            "    version         INT             NOT NULL,\n"
```

Comment:
You didnt update the database test files to deal with this change

Reply:
All test files that are present on the main branch have been updated.

Reply:
Missing test files have been added and updated now